configCivcraft.yml

```diff
@@ -3864,15 +3861,15 @@ recipes:
   Smelt_Coal_Ore_Expert:
     type: PRODUCTION
     name: Smelt Coal Ore
-    production_time: 50s
+    production_time: 64s
     input:
       coal ore:
         material: COAL_ORE
         amount: 256
     output:
       coal:
         material: COAL
-        amount: 640
+        amount: 768
```

Comment:
Pearling is like 1% of what coal is used for. The increase is really just to make the progression logical, 2x,3x etc.